packages/fetch-api/src/constants.ts

```diff
@@ -3,7 +3,7 @@ const gl = ((typeof window !== 'undefined' ? window : global) || {}) as any
 
 // GIPHY_API_URL is made to override the api url in testing environments
 // must be specified before the packages load
-export let serverUrl = gl.GIPHY_API_URL || 'https://api.giphy.com/v1/'
+export let serverUrl = gl.GIPHY_API_URL || 'https://api.giphy.com/'
```

Comment:
This change is necessary to support the new `v2/emoji/*` endpoints. However, it may cause some tests with this parameter to fail, so these tests should be updated.

Reply:
@pshoniuk this would break giphy.com since it would then request `https://api.giphy.com/v1/v1` unless we update all of our env variables in coordination with the sdk update. The coordination could easily go wrong 

Is it possible to move everything to v2?

Reply:
@giannif  I don't think it is possible to move everything to v2. We only have these new emoji endpoints placed in v2. Also, these new `v2/emoji/*` endpoints do not replace the old `v1/emoji` endpoint. They are both valid.

I think in the future, we may have more endpoints that work on different version prefixes. Currently, I see two options: 
 - We could create a new API client, such as `GiphyFetchV2`, that works with v2. But this might confuse end-users about when to use one client and when to use the other.
- In the second option, we can move the prefix directly into the API call, as I did in this PR. But, as you mentioned, this requires work with changing env variables. For this case, I think we can temporarily add something like this to keep the current behavior intact and report where we need to update the env variables:
```typescript
const oldPrefixRegex = /\/v1\/?$/;
if (oldPrefixRegex.test(serverUrl)) {
  console.warn('Some waring')
  serverUrl = serverUrl.replace(oldPrefixRegex, '/')
}
```

What do you think about it? Maybe you have other ideas?

Reply:
@pshoniuk thanks for digging in there. When I set this up, I definitely thought the api versions would move in unison. Still not sure why that's not the case, but I have another idea that should give us flexibility without risking breaking anything as it's all internal:

```typescript
function request(
    url: string,
    normalizer: (a: any) => any = identity,
    noCache: boolean = false,
    apiVersion = 1
) {
// not tested
    url = url.replace(/v(\d)/, `v${apiVersion}`)
```

It would be nice to move arguments 2,3, and 4 into an options object. Maybe that's too big of a change. Can we

Reply:
I like this idea, will implement it soon